### ISSUE TITLE
A11y_.Net Core_Winforms_MainForm_Dialog Controls_Font Dialog_Screen reader :  Narrator is not announcing the Labels Names "Effects" and "Sample"  present in the Font dialog Window

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/winforms#9229]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

Yes

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#Benchmark;#Narrator;#NVDA;

### Actual:
The Narrator screen reader is not announcing the Label Names "Sample" And "Effects" when we go by SCAN ON mode of Narrator.

Note : This issue is also reproducing with NVDA screen reader. NVDA is announcing the "Effects" Label but not the "Sample" Label

### Expected:
The Narrator/NVDA screen reader should announce the Labels Name while we Navigate through screen reader.

### User Impact:
Screen reader user will face difficulty in knowing the Label name of the section.

### Steps to reproduce

1. Install and Open Winforms App
2. Main Form screen would appear.
3. Tab till "Dialog Controls : Testing the Controls Under the Dialog Tab" and Hit enter.
4. Navigate to "Font Dialog" Button and Hit enter.
5. Observe with Screen reader that the Label Names are not announced by Narrator/NVDA.

### Attachment:

Narrator Attachment:
[Narrator Not announcing as Label.zip](https://github.com/dotnet/winforms/files/11659160/Narrator.Not.announcing.as.Label.zip)

NVDA Attachment:
![NVDA Label](https://github.com/dotnet/winforms/assets/85863895/65ad6954-0c58-432a-900e-a1cf2b84535f)

**Comment by John-Qiao** [comment]
This issue same as [5314](https://github.com/dotnet/winforms/issues/5314). In comments, @Tanya-Solyanik said that's a Windows issue and don't own content of this dialog.

**Comment by merriemcgaw** [comment]
This is the CommonDialogs and we can't fix it. The [Windows Bug](https://microsoft.visualstudio.com/OS/_workitems/edit/39833818) has been resolved as Won't Fix. We have an exception for some of the CommonDialogs already, I'll see what we need to do to get this closed.

**Comment by Tanya-Solyanik** [comment]
@John-Qiao - could you please remove this dialog from our accessibility test app if it hadn't been removed already.

Did the testers pick up this app from your file share rather than build the latest?

**Comment by John-Qiao** [comment]
@Tanya-Solyanik Ok, we will remove the font dialog from test app and rebuild it. We provide the build latest exe to testers.

**Comment by John-Qiao** [comment]
We have updated the test app that removed the font dialog.

**Comment by Ashutoshsahoo164** [comment]
@John-Qiao Can you please share the latest exe file to verify these bugs?

**Comment by Tanya-Solyanik** [comment]
@John-Qiao  - please coordinate with @merriemcgaw on how to provide a new test application.

**Comment by John-Qiao** [comment]
@Ashutoshsahoo164 you can test using this exe file: [Accessibility_Core_8.0_AppExe.zip](https://github.com/dotnet/winforms/files/11755595/Accessibility_Core_8.0_AppExe.zip)

@Tanya-Solyanik ok, we will merge these changes to our repo.

**Comment by Ashutoshsahoo164** [comment]
Hi @John-Qiao is the bug fixed?

**Comment by John-Qiao** [comment]
@Ashutoshsahoo164 we removed that font dialog from testing app. And updated the testing app in https://github.com/dotnet/winforms/commits/main/src/System.Windows.Forms/tests/AccessibilityTests.

**Comment by MelonWang1** [comment]
Verified this issue in the latest .NET 8.0 SDK build: 8.0.100-preview.7.23323.19 on winforms repo,  the font dialog already removed. Screenshot as below:
![9229](https://github.com/dotnet/winforms/assets/94418985/66642d4c-a7f5-4fad-8c87-444fefede322)

**Comment by MandiMan** [comment]
Verified this on .NET 8.0 TP build: 8.0.100-preview.6.23327.6, the font dialog already removed, result is same as above test result.

**Comment by Ashutoshsahoo164** [comment]
i have verified the environment , the font dialog is now removed. Hence bug is closed

GitHubTags:#Closed;
